dev/inventory/defaults.yaml

```diff
@@ -107,16 +111,17 @@ volumes_defaults:
 node_defaults:
   vcpu: 2
   memory: 4096
-  network_interfaces:
-    - mode: macvtap
-      target: enp42s0
-      mtu: 1500
-      dhcp4: true
-      dhcp6: false
+  network_interfaces: []
```

Comment:
why did we remove the macvtap by default?

Reply:
I was testing the airgapped setup.